Section_II/competition_and_trophies.tex

```diff
@@ -274,53 +274,107 @@
       (e.g 'Pushing Red - Direct Free-Kick Blue')
-\item The assistant referee who operates the game controller clicks on "Direct / Indirect Free Kick" Blue/Red.
-\item The referee places the ball depending on the call and announces "Free Kick Ready".
+\item \removed{The assistant referee who operates the game controller clicks on
+    "Direct / Indirect Free Kick" Blue/Red.}
+  \added{The referee communicates the game interruption and to which team it is
+    awarded to the game controller.}
+\item \removed{The referee places the ball depending on the call and announces "Free Kick Ready".
       Until the referee announced the free kick to be ready robots may move
       their head to track the ball but must otherwise remain stationary.
-      Robots which are significantly moving during this phase are removed from the field.
-\item The assistant referee who operates the game controller clicks on
-      Prepare Direct / Indirect Free Kick" Blue/Red.
-      The player taking the free kick has up to 30 seconds to position
-      themselves for the free kick.
-      The team taking the free kick may announce that the player is ready to
-      take the free kick at any point.
+      Robots which are significantly moving during this phase are removed from the field.}
+      \added{%
+        The robots are allowed 5 seconds to stand-up and stop any infringement
+        to the rules.
+        At the end of these 5 seconds, the referee places the ball according to
+        the game interruption.
+        In case a robot is located at a distance lower than $D$, the following
+        actions are performed in order:
+        \begin{enumerate}
+        \item Move all penalized robots at a distance lower than $D$ of the ball
+          away, placing them according to the removal penalty rule so that they
+          can finish serving their penalty somewhere else.
+        \item Apply a removal penalty on all fallen robots at a distance lower
+          than $D$ from the ball.
+        \item If previous steps were not sufficient to ensure distance $D$, the
+          referee tries to find a nearby location for the ball that is at a
+          distance of at least $D$ from all objects.
+          This location should respects the rule for the game interruption.
+          In particular, there are no alternative location for penalties, goal
+          kicks and corner kicks.
+        \item If previous steps were not sufficient, the remaining robots
+          at a distance of less than $D$ from the ball location are moved away
+          from the ball, ensuring that they are at least at a distance of $D$
+          from all objects, including the ball.
+        \end{enumerate}
+      }
+\item \removed{The assistant referee who operates the game controller clicks on
+        "Prepare Direct / Indirect Free Kick" Blue/Red.}
+      \added{%
+        The referee communicates that the Game Interruption is ready to the
+        GameController.%
```

Comment:
Add: "or the assistant referee operating the game controller in a physical competition."